commit 4a848fb430e57c633ca302ac613da5e3f21ade8d
Author: Hubert Figuière <hub@figuiere.net>
Date:   Tue Nov 19 22:44:04 2019 -0500

    Fix gtk2 detection

diff --git a/muse3/CMakeLists.txt b/muse3/CMakeLists.txt
index 51331209..1e5aaa92 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -490,7 +490,7 @@ if (ENABLE_LV2)
     if (LILV_FOUND AND SORD_FOUND AND LV2_FOUND)
         ## find Gtk2
         if(ENABLE_LV2_GTK2)
-          find_package(GTK2 COMPONENTS gtk gtkmm)
+          pkg_check_modules (GTK2 gtk+-2.0 gtkmm-2.4)
           if(GTK2_FOUND)
               set(HAVE_GTK2 ON)
           endif(GTK2_FOUND)
